sections/data_release_production_pipelines.tex

```diff
@@ -71,26 +75,332 @@ \section{Data Release Production}
+
+\paragraph{MatchSemiBlind}
+\label{sec:drpBootstrapImChar_MatchSemiBlind}
+
+Delegate to the \hyperref[sec:acSingleVisitReferenceMatching]{Single Visit Reference Matching algorithmic component} to match source catalogs to a global reference catalog.  This occurs over the full field of view, ensuring robust matching even when some CCDs have no matchable stars due to crowding, flux limits, or artifacts.
+
+``Semi-Blind'' refers to the fact that the WCS is not yet well known (all we have is what is provided by the observatory), so the matching algorithm must account for an unknown (but small) offset between the WCS-predicted sources positions and the reference catalog positions.
+
+\paragraph{SelectStars}
+\label{sec:drpBootstrapImChar_SelectStars}
+
+Use reference catalog classifications and source flags to select a clean sample stars to use for later stages.
+
+If we decide not to rely on a pre-existing reference catalog to separate stars from galaxies and other objects, we will need a new algorithmic component to select stars based on source measurements.
+
+\paragraph{FitWCS}
+\label{sec:drpBootstrapImChar_FitWCS}
+
+Delegate to the \hyperref[sec:acSingleVisitAstrometricFit]{Single Visit Astrometric Fit algorithmic component} to determine the WCS of the image.
+
+We assume this works by fitting a simple mapping from the visit's focal plane coordinate system to the sky and composing it with the (presumed fixed) mapping between CCD coordinates and focal plane coordinates.  This fit will be improved in later pipelines, so it does not need to be exact; $<$0.05 arcsecond accuracy should be sufficient.
+
+As we iterate in crowded fields, the number of degrees of freedom in the WCS should be allowed to slowly increase.
+
+\paragraph{FitPSF}
+\label{sec:drpBootstrapImChar_FitPSF}
+
+Delegate to the \hyperref[sec:acFullVisitPSF]{Full Visit PSF Modeling algorithmic component} to construct an improved PSF model for the image.
+
+Because we are relying on a reference catalog to select stars, we should be able to use colors from the reference catalog to estimate SEDs an include wavelength dependence in the fit.  If we do not use a the reference catalog early in BootstrapImChar, PSF estimation here will not be wavelength-dependent.  In either case the PSF model will be further improved in later pipelines.
+
+PSF estimation at this stage must include some effort to model the wings of bright stars, even if this is tracked and constrained separately from the model for the core of the PSF.
+
+As we iterate in crowded fields, the number of degrees of freedom in the PSF model should be allowed to slowly increase.
+
+\paragraph{WriteDiagnostics}
+\label{sec:drpBootstrapImChar_WriteDiagnostics}
```

Comment:
I'm wondering what the motivation in calling this out specifically is. I can see that it falls at a point in the processing where one might well well want diagnostics, but it's not obvious that the same doesn't apply elsewhere.